src/Tribe/RSVP/Service_Provider.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Tribe\Tickets\RSVP;
+
+use Tribe\Tickets\RSVP\Early_Access\Assets;
+use Tribe\Tickets\RSVP\Early_Access\Early_Access;
+use Tribe\Tickets\RSVP\Early_Access\Template;
+
+/**
+ * Class Service_Provider
+ *
+ * @since TBD
+ *
+ * @package Tribe\Tickets\RSVP
+ */
+class Service_Provider extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * @inheritDoc
+	 *
+	 * @since TBD
+	 */
+	public function register() {
+		$this->register_early_access();
+	}
```

Comment:
Check the pattern we used on Views V2 with Hooks, Assets, and registration of the Service Provider.
https://github.com/moderntribe/the-events-calendar/blob/master/src/Tribe/Views/V2/Service_Provider.php